client/components/videos-ui/index.jsx

```diff
@@ -38,7 +38,7 @@ const VideosUi = () => {
 							</li>
 							<li>
 								<Gridicon icon="checkmark" size={ 12 } />{ ' ' }
-								{ translate( 'Familiarize yourself with Wordpress' ) }
+								{ translate( 'Familiarize yourself with WordPress' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.